packages/sanity/src/core/studio/tasks/components/list/styles.ts

```diff
@@ -0,0 +1,11 @@
+import {Card, CardProps} from '@sanity/ui'
```

Comment:
Just curiosity, why are we moving this to our own implementation of styled component if it's not changing any style yet.